run.py

```diff
@@ -20,14 +22,24 @@
 
 @main_router.message(Command(commands=["start", "help"]))
 async def start(message: types.message):
+    user_telegram_id = message.chat.id
+    user_telegram_username = message.from_user.username
+
```

Comment:
Why do you move lines up and down unnecessarily?

Reply:
see previous comment